portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsColorField.testcase

```diff
@@ -131,14 +131,41 @@ definition {
 	}
 
 	@description = "Verify that the Field Label can be changed"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test LabelCanBeEdited {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-133140 LabelCanBeEdited pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Color");
+
+		FormFields.viewTextLabel(
+			fieldLabel = "Color",
+			fieldName = "Color");
+
+		Form.editFieldLabelMetal(fieldValue = "Test Color");
+
+		FormFields.viewTextLabel(
+			fieldLabel = "Test Color",
+			fieldName = "Color");
+
+		Form.save();
+
+
+		Form.publishForm();
+
+		FormsAdminNavigator.gotoPublishedForm();
 
+		FormViewBuilder.validateFieldLabel(fieldLabel = "Test Color");
```

Comment:
@leilanyaragao Is it possible to use the 
```
FormFields.viewColorLabel(...);
```
instead?  `FormViewBuilder ` belongs to App Builder, not Forms.

Thanks!